.eslintrc.js

```diff
@@ -228,6 +234,10 @@ module.exports = {
           "The 'use client' pragma can't be used with export * in the same module. This is not supported by Next.js.",
         selector: 'ExpressionStatement[expression.value="use client"] ~ ExportAllDeclaration',
       },
+      {
+        message: 'Avoid calling Error without `new`. Use `new Error()` instead.',
```

Comment:
```suggestion
        message: 'Do not call Error without `new`. Use `new Error()` instead.',
```

To be more consistent with the other messages.